data/map systems.txt

```diff
@@ -2110,6 +2109,63 @@ system Acnanjo
 		distance 3314.9
 		period 1199.60
 
+system Acrab
+	pos -297.08377 146.935
+	government Syndicate
+	link Fomalhaut
+	link Markab
+	link Tabit
+	asteroids "small rock" 40 4.224
+	asteroids "large rock" 32 6.27
+	asteroids "medium metal" 9 5.61
+	asteroids "large metal" 4 4.158
+	minables aluminum 3 10.167916
+	minables iron 7 9.6900905
+	fleet "Large Core Merchants" 800
```

Comment:
```suggestion
	fleet "Small Core Merchants" 800
	fleet "Large Core Merchants" 1000
	fleet "Small Syndicate" 1200
	fleet "Large Syndicate" 2000
	fleet "Human Miners" 3000
	fleet "Republic Logistics" 3600
```
Non-pirate fleet frequencies based on neighboring Fomalhaut and Markab. Pirate frequency should probably be lower (it's 0 in both Fomalhaut and Markab) but not necessarily zero, because of the route via Tabit.